src/bundle/Controller/ContentTypeController.php

```diff
@@ -523,9 +530,49 @@ public function updateAction(
             'content_type' => $contentTypeDraft,
             'form' => $form->createView(),
             'language_code' => $baseLanguage ? $baseLanguage->languageCode : $language->languageCode,
+            'field_type_toolbar' => $this->fieldTypeToolbarFactory->create(),
         ]);
     }
 
+    public function addFieldDefinitionFormAction(
+        Request $request,
+        ContentTypeGroup $group,
+        ContentTypeDraft $contentTypeDraft,
+        string $fieldDefinitionIdentifier,
+        ?Language $language = null,
+        ?Language $baseLanguage
```

Comment:
Incorrect order of optional & mandatory arguments

```suggestion
        ?Language $language = null,
        ?Language $baseLanguage = null
```